modules/cluster-logging-deploy-cli.adoc

```diff
@@ -249,60 +183,35 @@ spec:
 <1> You must specify the `openshift-logging` namespace.
 <2> Specify `stable`, or `stable-5.<x>` as the channel.
 <3> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the CatalogSource object you created when you configured the Operator Lifecycle Manager (OLM).
-+
-[source,terminal]
-----
-$ oc create -f <file-name>.yaml
-----
-+
-For example:
-+
-[source,terminal]
-----
-$ oc create -f olo-sub.yaml
-----
-+
-The Red Hat OpenShift Logging Operator is installed to the `openshift-logging` namespace.
 
-.. Verify the Operator installation.
-+
-There should be a Red Hat OpenShift Logging Operator in the `openshift-logging` namespace. The Version number might be different than shown.
+.. Apply the `Subscription` object by running the following command:
 +
 [source,terminal]
 ----
-$ oc get csv -n openshift-logging
+$ oc apply -f <filename>
 ----
 +
-.Example output
-[source,terminal]
-----
-NAMESPACE                                               NAME                                         DISPLAY                  VERSION               REPLACES   PHASE
-...
-openshift-logging                                       clusterlogging.5.1.0-202007012112.p0         OpenShift Logging          5.1.0-202007012112.p0              Succeeded
-...
-----
-
-. Create an OpenShift Logging instance:
+The {clo} is installed to the `openshift-logging` namespace.
 
-.. Create an instance object YAML file (for example, `olo-instance.yaml`) for the Red Hat OpenShift Logging Operator:
+. Create a `ClusterLogging` custom resource (CR):
 +
 [NOTE]
 ====
-This default OpenShift Logging configuration should support a wide array of environments. Review the topics on tuning and
-configuring {logging} components for information about modifications you can make to your OpenShift Logging cluster.
+This default `ClusterLogging` CR configuration should support a wide array of environments. Review the topics on tuning and
```

Comment:
```suggestion
This default `ClusterLogging` CR configuration should support a wide array of environments. Review the topics on tuning and configuring {logging} components for information about modifications you can make to the {logging}.
```
Perhaps all on one line while you're here? It looks nice in the preview, but the line break seems odd.